content/zh-tw/devops.md

```diff
@@ -0,0 +1,21 @@
+---
+title: DevOps
+status: Completed
+category: 概念
+tags: ["方法論", "", ""]
+---
+
+## 什麼是 DevOps {#what-it-is}
+
+DevOps 代表著由團隊主導應用程式開發（development）到正式維運（operations）完整過程的一種方法，所以它才被稱作 DevOps。DevOps 不只是一套技術的運用，更是文化與流程的全面轉變。DevOps 要求工程團隊專注在更小的元件，而非一次處理整個功能，並且減少交接的次數，因為頻繁的交接正是導致錯誤的常見原因。
+
+## 解決的問題 {#problem-it-addresses}
+
+<!-- Need to change the links after /zh-tw/tightly-coupled-architectures and /zh-tw/monolithic_apps are completed -->
+傳統上，在具有[緊耦合](/tightly-coupled-architectures/)[單體式應用程式](/monolithic_apps/)的複雜組織裡，工作通常被分散在不同的團隊之間。這導致工作需要多次的交接與漫長的前置時間（lead time）。每當一個元件或更新準備就緒時，它就會被放到另一個團隊的工作佇列中。由於每個人都只處理專案的一小部分，而這種方式會導致主導權的缺乏。他們的目標將會是把工作交給下一個團隊，而不是為客戶提供正確的功能，這顯然是本末倒置。
```

Comment:
```suggestion
傳統上，在具有[緊耦合](/zh-tw/tightly-coupled-architectures/)[單體式應用程式](/zh-tw/monolithic_apps/)的複雜組織裡，工作通常被分散在不同的團隊之間。這導致工作需要多次的交接與漫長的前置時間（lead time）。每當一個元件或更新準備就緒時，它就會被放到另一個團隊的工作佇列中。由於每個人都只處理專案的一小部分，而這種方式會導致主導權的缺乏。他們的目標將會是把工作交給下一個團隊，而不是為客戶提供正確的功能，這顯然是本末倒置。
```

Reply:
Is it ok to link to zh-tw page before these two are not ready?

Reply:
Per [discussion](https://cloud-native.slack.com/archives/C055KERTNCX/p1684726776293839?thread_ts=1683800025.642469&cid=C055KERTNCX), just linking to zh-tw/{words} directly.